test/role_test.lua

```diff
@@ -70,7 +70,7 @@ function g.test_yaml()
 
     t.assert_str_matches(call('get_clusterwide_schema_yaml'), '## Example:\n.+')
     t.assert_equals(call('get_clusterwide_schema_lua'), {spaces = {}})
-    t.assert_equals(get_section(), '')
+    t.assert_equals(call('get_clusterwide_schema_yaml'), get_section())
```

Comment:
Does it make sense to check example explicitly?

Reply:
Before this patch, the example was returned by `get_clusterwide_schema_yaml`, and the section was empty. Now I check that they're equal